bpmsuite/bpmsuite70-businesscentral-monitoring-mysql.json

```diff
@@ -1,678 +0,0 @@
-{
```

Comment:
I am not sure this should be deleted as whole.
There is similar template available (bpmsuite70-businesscentral-monitoring.json), but that one contains also smart router. I think it should be renamed.
This template could be kept, should contain only Runtime monitoring.

Reply:
I see. The current naming seems pretty confusing. I know naming things is hard, but we need to make sure the name reflects the content. So `bpmsuite70-businesscentral-monitoring` should probably only contain the BC monitoring and we need another template which would contain both the BC monitoring app and smart router. Perhaps `bpmsuite70-businesscentral-monitoring-with-smartrouter`? I now the names are getting longer and longer, but not sure we can do much to make it shorter and at the same time still clear what the template is about.

Reply:
Agreed with removing the DB stuff, and renaming files to match what's actually in them.

Aside, @psiroky and @sutaakar : Please note that a bpmsuite70-1.0.Beta01 tag was added to the images/templates we had ready for initial handover, with the one addition of changing the 1.0 image references to 1.0.Beta01. Since that time, I have reverted back to 1.0, merged in master to bpmsuite-wip, and also updated the image streams file in how it refers to the docker images (using a "from" element instead of a "dockerImageRepository" element). tl;dr: please rebase any work you have against the latest in bpmsuite-wip and update your PRs. Thanks!